README.md

```diff
@@ -94,55 +99,67 @@ Several features are available at the surveypro instance level for different are
 - Assign an indent to questions to make parent-child relations clearer;
 - Check the effectiveness of parent-child relations;
 - Share in-progress surveypro definitions in XML with partners;
-- Option to preserve history (allow the users to change their response BUT forcing them to save the modified response to a new one in order to preserve the history of what they do?);
-- Keep "In progress" responses (do not drop "in progress" responses even if they should be dropped due to surveypro policies);
+- Option to preserve history (allow the users to change their response BUT forcing them to save the modified response to a new one in order to preserve the history of what they did.);
+- Automatic drop of abandoned responses.
+- Keep "In progress" responses (do not drop "in progress" responses even if they should be, because of surveypro policies);
+- Split a surveypro across multiple pages;
 - Custom style sheet;
 - Option for anonymous responses;
 - Mandatory or optional questions;
 - Notes (filling instructions) for each item;
 - Format of short text items (email, URL, or custom regular expressions);
 - Assign a name and custom number to each question;
-- Define a custom question number;
 - Tool to reorder items;
-- Hide items still in progress or not ready to show.
-- Custom thanks webpage.
-- Tool to duplicate an item.
-- Possibility to create relational databases once answers are exported.
-- Limit the minimum and maximum year at surveypro level.
-- Limit the input range of each item at the item level.
-- Use "classic" surveys like COLLES, ATTLS, Critical Incidents.
-- Have COLLES built using radio buttons or select items.
-- Automatic drop of abandoned responses.
-- Choose whether the label or value of checkboxes and selects should be in the response export file.
-- Get positional answers of ticks in checkbox questions in the response export file.
-- Mark items for form population to search among submitted responses.
-- Have a surveypro from a mastertemplate offering the multilanguage feature.
+- Hide items still in progress or not ready to show;
```

Comment:
What are all these semicolons (`;`) that you're adding everywhere instead of full stops (`.`)? They look super strange to me... are you sure that they are correct English? (I'm not).

Reply:
no, they are correct in Italian when you make a list. Each line ends with a semicolon and only the last ends with a dot.
To be honest: I do not know if it is correct in English.

About the bricks suggestion, it is probably correct.
I did not know about the possibility to add a folder and divide all the content that I saved in just one document over more files.

At the moment, please, merge this. I will divide this release into more files once I get an idea about "how to make enlighth this readme" and "how to classify all the same tons of information" dividing them in more files.

Reply:
anyway, I changed semicolons in dots to be safer